README.md

```diff
@@ -35,7 +35,7 @@ To test Checkout:
 ## Getting started
 
 See the
-[Vipps MobilePay Getting started guide](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/vipps-getting-started)
+[Vipps MobilePay Getting started guide](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/getting-started)
```

Comment:
is this link correct? 
![image](https://user-images.githubusercontent.com/110486952/225931442-7e51d93a-9be0-4c64-bf7f-1fce34baf18a.png)

Reply:
I'm merging them all at once

Reply:
So, it's not there now, but will there soon.